### DISCUSSION TITLE
ent vs gorm

### DISCUSSION THREAD

**Discussion by kuchaguangjie** [suyuan32/simple-admin-core#240] [General]
1. why use ent over gorm?
2. how to use gorm instead of ent ?

**Comment by kuchaguangjie** [comment]
propose:
1. let `goctls` able to generate gorm code, just like it can generate ent code.
2. let user choose to use gorm or ent when generate rpc & api code.

**Comment by suyuan32** [comment]
Support for gorm code generation is not considered for the time being. You can wait for the community to contribute code to goctls to add related support.
